doc/api/cli.md

```diff
@@ -730,6 +730,65 @@ Export keyword before a key is ignored:
 export USERNAME="nodejs" # will result in `nodejs` as the value.
 ```
 
+### `--env-file-override-local`
+
+> Stability: 1.1 - Active development
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Override existing environment variables on your machine with values specified in
+your environment files.
+
+```bash
+node --env-file=.env --env-file-override-local index.js
+```
+
+Example:
+
+Assume you have a local environment variable on your machine
+
+```bash
+API_KEY="local-key"
+```
+
+Meanwhile, your `.env` file contains:
+
+```bash
+API_KEY='env-custom-key'
+```
+
+If you want to use the value from the `.env` file, you should add the
+`--env-file-override-local` flag when running your application, or use
+[`process.loadEnvFile({ override: true })`][] in your code.
```

Comment:
This line is unnecessary for the CLI flag documentation. You should also mention how --env-file flag is required as well